web/carousel/src/screens/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Home from "./Home";
+import Login from "./Login";
+//import Header from '../components/Header'
+
+const App = () => {
+  return (
+    <>
+      {/* <Header /> */}
+      <Router>
+        <Switch>
+          <Route path="/" exact component={Home} />
+          <Route path="/login" exact component={Login} />
+        </Switch>
+      </Router>
```

Comment:
Router for Pages is defined here. We can also add `Header` component after we implement it in components/Header